### PR TITLE
added-airtableapp

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/airtable.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+id: airtable
+title: Sumo logic App for Airtable
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/airtable.md:3]:
```suggestion
title: Sumo Logic App for Airtable
```

File: docs/integrations/saas-cloud/airtable.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+sidebar_label: Airtable
+description: The Sumo Logic App for the Airtable App offers functionality for monitoring and analyzing your organization's Airtable audit logs.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/airtable-icon.png')} alt="airtable-icon.png" width="150" />
+
+The Airtable App for Sumo Logic monitors and analyses your organization's Airtable audit logs, providing insights into user activity, data trends, and security events. This app is based on the Airtable Audit Logs, which provide detailed information on all actions performed in your Airtable account.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/airtable.md:12]:
```suggestion
The Airtable App for Sumo Logic monitors and analyzes your organization's Airtable audit logs and provides insights into user activity, data trends, and security events. This app is based on the Airtable Audit Logs, which provide detailed information on all actions performed in your Airtable account.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Changed UK to US spelling, capitalized Sumo Logic, and a couple other minor edits

File: docs/integrations/saas-cloud/airtable.md (hunk 2 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/airtable.md:12]: @kimsauce Oh! thank you "analyzes" needs to be fixed, but the second suggestion will not apply here. 
These two are independent clauses and when two independent clauses are related in meaning and are closely connected, a conjunction is not always necessary to join them. Instead, a comma can be used to separate the two clauses. This is known as a comma splice.

File: docs/integrations/saas-cloud/airtable.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Before you begin, collect logs from Airtable API and ingest them into Sumo Logic. Refer to the [Airtable API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/airtable-source/) to create the source and use the same source category while installing the app.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **Airtable App** using the steps described in the [Airtable Cloud-to-Cloud Source](docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/airtable-source/). If you already have set up your data, skip this step by clicking on **Next**.
+1. Complete the following fields:
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/airtable.md:84]:
```suggestion
      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For example, `_sourceCategory=MyCategory`.
```

File: docs/integrations/saas-cloud/airtable.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+| Json "enterprise_account_id", "originating_user_id", "api_name", "api_version", "action_id", "client.ipaddress", "request.requestid", "request.starttime", "request.modelclassname", "request.modelid", "request.action", "response.success" as enterprise_account_id, originating_user_id, api_name, api_version, action_id, ipaddress, requestid, starttime, modelclassname, modelid, action, success nodrop
+| where api_name matches "{{api}}" or isNull(api_name)
+| where modelclassname matches "{{model}}" or isNull(modelclassname)
+| where action matches "{{action}}" or isNull(action)
+| count_distinct (action_id)
+```
+
+## Installing the Airtable App
+
+Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
```
Comment (stacykor) [inline docs/integrations/saas-cloud/airtable.md:74]:
```suggestion
Locate and install the app from the **App Catalog**. To see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
```

**Review by stacykor — CHANGES_REQUESTED** [review]

**Pull request merged**